test/git/test_invocations.py

```diff
@@ -397,7 +401,7 @@ def make_setuptools_wheel_with_pip(self):
 
     def make_binary_wheelname(self, app):
         return "%s-2.0-%s-%s-%s.whl" % (app,
-            "".join([impl, impl_ver]), abi, plat)
+            "".join([impl, impl_ver]), abi, plat.replace("-", "_"))
```

Comment:
The issue seems to be that sometimes `impl_ver` is 372.

```
FAIL: test_extension_wheel_setuptools (__main__.SetuptoolsUnpacked)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/git/test_invocations.py", line 853, in test_extension_wheel_setuptools
    self.assertIn(created, glob(os.path.join(wheelhouse, "*.whl")))
AssertionError: '/tmp/tmpchzmptu6/demoappext-setuptools-repo/dist/demoappext-2.0-pp36-pypy3_72-linux_x86_64.whl' not found in ['/tmp/tmpchzmptu6/demoappext-setuptools-repo/dist/demoappext-2.0-pp372-pypy3_72-linux_x86_64.whl']
```

Reply:
That's the same error. One or multiples of setuptools/pip/packaging is out of date for PyPy >=7.3.1 and causing wrong ABI tag to be generated. Will try updating them in tox - cannot reproduce locally with updated versions.

Reply:
#224 is working now.